app/views/home/index.html.erb

```diff
@@ -4,4 +4,9 @@
 <% else %>
   <%= link_to "login", new_user_session_path %>
 <% end %>
-<h1><%= link_to "Games", "/games" %></h1>
\ No newline at end of file
+<br><br>
+<label for="games">Games List</label>
+<%=form_tag('/home/0/select_game') do %>
+  <%= select_tag(:game, options_for_select(@games.map{ |i| i.name })) %>
+  <%= submit_tag "Select" %>
+<% end %>
```

Comment:
`form_with` is preferrable over `form_tag`. There is also a `collection_select` method that does what you want. Something like:
```suggestion
<%= form_with url: select_game_path do |f| %>
  <%= f.collection_select :game, Game.all, :id, :name, prompt: true) %>
  <%= f.submit "Select" %>
<% end %>
```

Reply:
Found it out when looking into the next issue, the form_tag is soft deprecated. Changed to your suggestion